xonsh/completer.py

```diff
@@ -78,6 +80,18 @@ def _normpath(p):
 
     return p
 
+def completionwrap(s):
+    """ Returns the repr of input string s if that string contains 
+    a 'problem' token that will confuse the xonsh parser
+    """
+    space = ' '
+    slash = '/'
```

Comment:
I'm thinking it might be the right time to put that into a function in `tools.py` as a function (`xonsh.tools.get_sep()` or similar?), since it's used a few places now.  We can then go through and replace the hard-coded versions with calls to that function.

Not sure if that belongs in this PR or if it should happen separately.